### PR TITLE
Skip puppeteer tests for now

### PR DESCRIPTION
**Pull request by wtgodbe**
https://github.com/dotnet/aspnetcore/issues/54176

### CODE REVIEW

**Review by amcasey — APPROVED** [review]

File: .azure/pipelines/components-e2e-tests.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ variables:
   value:  AspNetCore
 - name: npm_config_cache
   value: src/submodules/Node-Externals/cache
+- name: PUPPETEER_SKIP_DOWNLOAD
+  value: 1
```
Comment (javiercn) [inline .azure/pipelines/components-e2e-tests.yml:31]: Puppeteer is only used by SignalR, so it's fine if its disabled here "forever".

File: eng/Npm.Workspace.nodeproj [hunk 2]
```diff
@@ -13,7 +13,8 @@
     <TestDependsOnAspNetRuntime>false</TestDependsOnAspNetRuntime>
     <ContainsFunctionalTestAssets>false</ContainsFunctionalTestAssets>
     <BuildHelixPayload>false</BuildHelixPayload>
-    <_NpmAdditionalEnvironmentVariables>npm_config_cache=$(RepoRoot)src/submodules/Node-Externals/cache</_NpmAdditionalEnvironmentVariables>
+    <!-- Skip puppeteer tests until https://github.com/dotnet/aspnetcore/issues/54176 is resolved -->
+    <_NpmAdditionalEnvironmentVariables>PUPPETEER_SKIP_DOWNLOAD=1;npm_config_cache=$(RepoRoot)src/submodules/Node-Externals/cache</_NpmAdditionalEnvironmentVariables>
```
Comment (javiercn) [inline eng/Npm.Workspace.nodeproj:17]: Not sure how this didn't break the SignalR tests in general. Afaik, the browsers are downloaded as a post-install step.

File: .azure/pipelines/components-e2e-tests.yml (hunk 1 above)
Comment (javiercn) [reply .azure/pipelines/components-e2e-tests.yml:31]: The functional tests do.

Comment (javiercn) [reply .azure/pipelines/components-e2e-tests.yml:31]: image

You use Karma, and that uses puppeteer

**Pull request merged**